File: jiaoyu-teacher-interface/src/main/java/com/jiaoyu/teacher/service/KegdService.java
package com.jiaoyu.teacher.service;
import java.util.List;
import com.jiaoyu.pojo.Kegd;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface KegdService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Kegd> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Kegd kegd);
	
	
	/**
	 * 修改
	 */
	public void update(Kegd kegd);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Kegd findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Kegd kegd, int pageNum,int pageSize);
	
}

File: jiaoyu-teacher-web/src/main/webapp/js/service/kegdService.js
//服务层
app.service('kegdService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../kegd/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../kegd/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../kegd/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../kegd/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../kegd/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../kegd/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../kegd/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
